python-lib/text_extraction/__init__.py

```diff
@@ -106,9 +113,22 @@ def extract_markdown_chunks(markdown, filename):
     header_stack = []
     initial_metadata = {}
 
+    in_code_block = False
+
     for line in lines:
         stripped_line = line.strip()
 
+        if stripped_line.startswith("```"):
```

Comment:
Actually the basic markdown syntax also creates code blocks by indenting with 4 spaces or tab, we should take this into account

Reply:
ok tab/4-spaces now handled